src/styles/hideScrollbar.css

```diff
@@ -1,3 +1,3 @@
-::-webkit-scrollbar {
+/* ::-webkit-scrollbar {
   display: none;
-}
+} */
```

Comment:
since the only content in the file is no longer used, should the file just be removed or..?

Reply:
I think it's better to just comment it out in case somebody wants to revert for some reason. But you're the senior dev; do you want me to remove the file?

Reply:
Ok lets go ahead and remove the file. The less unused, commented code there is in the codebase, the easier it will be to maintain. Also, it looks like there are 4 other files that import `hideScrollbar.css`. Unless you think otherwise, I think those imports should be removed as well. If somebody wants to re-hide the scrollbar, it shouldn't be difficult to add the styling back in for it.